pom.xml

```diff
@@ -61,7 +61,7 @@
         <dep.docker-java.version>3.2.5</dep.docker-java.version>
         <dep.joda.version>2.10.6</dep.joda.version>
         <dep.coral.version>1.0.12</dep.coral.version>
-        <dep.confluent.version>5.4.3</dep.confluent.version>
+        <dep.confluent.version>5.5.0</dep.confluent.version>
```

Comment:
How about the latest version 6.1 ?

Reply:
I updated to latest 5.x.x, I don't want to update to 6.1 just yet.